converter/internal/prometheusconvert/prometheusconvert.go

```diff
@@ -154,7 +154,7 @@ func appendServiceDiscoveryConfigs(pb *prometheusBlocks, serviceDiscoveryConfig
 			exports = appendDiscoveryLightsail(pb, common.LabelWithIndex(labelCounts["lightsail"]-1, label), sdc)
 		}
 
-		targets = append(exports.Targets, targets...)
+		targets = append(targets, exports.Targets...)
```

Comment:
This fixes a backwards ordering that was not a functional problem but weird. Discovered when leveraging this for promtail.